test/base/testing_brave_browser_process.cc

```diff
@@ -95,6 +95,12 @@ TestingBraveBrowserProcess::https_upgrade_exceptions_service() {
   return nullptr;
 }
 
+webcompat_exceptions::WebcompatExceptionsService*
+TestingBraveBrowserProcess::webcompat_exceptions_service() {
+  NOTREACHED();
```

Comment:
NOTREACHED_NORETURN

Reply:
I think I need to return nullptr here because the return type is not void.

Reply:
Actually I changed the return type to void and used NOTREACHED_NORETURN.